### DISCUSSION TITLE
Error building webpack "cache_springs is already renamed to e, but it's renamed as a"

### DISCUSSION THREAD

**Discussion by Kahor** [webpack/webpack#18732] [Q&A]
Currently using SWC (to replace Babel) in our project, I updated package.json to use latest version of swc/core (1.7.22, from 1.4.2) but I am now getting the following error when building webpack (didn't get this error before).
```
uiAssets/app.chunk.753.cd0a71a41f908fa784ad.js from Terser plugin
failed to handle: internal error: entered unreachable code: cache_springs is already renamed to e, but it's renamed as a
Error: failed to handle: internal error: entered unreachable code: cache_springs is already renamed to e, but it's renamed as a
```
I tried to update webpack-terser-plugin from 5.3.6 to 5.3.10 but this still fails.
Currently using webpack 5.93.0 

Is that a known issue somehow ? Any idea ?
Thanks !

**Comment by alexander-akait** [comment]
Sounds like bug in `swc/core`, can you provide your configuration?
